spec/controllers/pull_requests_controller_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe PullRequestsController, type: :controller do
+  describe "GET #index" do
+    let(:organization1) { create(:organization, login: 'my_organization') }
+    let(:organization2) { create(:organization, login: 'other_organization') }
+    let(:repo1) { create(:repository, organization: organization1) }
+    let(:repo2) { create(:repository, organization: organization2) }
+    let!(:pr1) { create(:pull_request, repository: repo1) }
+    let!(:pr2) { create(:pull_request, repository: repo2) }
+    let!(:user1) { create(:github_user, login: "user_login") }
+    let!(:github_session) { instance_double("GithubSession") }
+
+    before do
+      allow(github_session).to receive(:token).and_return("token")
+      allow(github_session).to receive(:user).and_return(user1)
+      expect(subject).to receive(:authenticate_github_user).and_return(true)
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
RSpec/NamedSubject: Name your test subject if you need to reference it explicitly.